installation/README.md

```diff
@@ -10,31 +10,31 @@ Download the repository as a zip or use the git clone command.
 - Download this repository as zip file
 - Connect to Nodegrid the WebUI as a admin user
 - Open the File Manager and navigate to admin_group
-- Upload the downloaded file `zpe.nodegrid_examples-main.zip` into the folder
+- Upload the downloaded file `Salt-main.zip` into the folder
 - Close the File Manager window
 - Open a Console connection to Nodegrid
 - Access the shell as an admin user using the `shell` command
-- Navigate to `/var/local/file_manager/admin_group/` 
+- Navigate to `/var/local/file_manager/admin_group/`
 ```shell script
 cd /var/local/file_manager/admin_group/
 ```
 - Extract the zip file with
 ```shell script
-mv zpe.nodegrid_examples-main.zip zpe.nodegrid_examples.zip
-unzip zpe.nodegrid_examples.zip
+unzip Salt-main.zip
+mv Salt-main Salt
```

Comment:
couldn't you remove this step by unziping directly to a folder named `Salt`? I think that the options `-j` and `-d` could help.

Reply:
It would be a nice enhancement but so far no unzip options allowed me to change the destination folder name.

the -d indeed specifies or creates a new directory, but inside that new dir, the original name is kept.